### PR TITLE
Reconnect frontend to backend

### PR DESCRIPTION
**Pull request by JayAlvinCVallescas**
Please fix this, as your lowly classmate has some skill issues

### CODE REVIEW

File: package-lock.json
Comment (H-isaac23) [inline package-lock.json:1]: Delete mo to, dapat walang package-lock.json na lalabas kapag ni-run mo `yarn install`

File: frontend-react/.env.example [hunk 1]
```diff
@@ -1 +1 @@
-VITE_URL_PREFIX=http://localhost
\ No newline at end of file
+VITE_URL_PREFIX=http://localhost:5173
```
Comment (H-isaac23) [inline frontend-react/.env.example:1]: Nag-pull ka diba? Hahahah wait attach ko sa trello yung .env na file

**Comment by H-isaac23** [comment]
Hanapin mo ganito na para sa login at signup, then palitan mo nung nasa trello
https://github.com/H-isaac23/CodeWars/blob/913e9dc7889fb58e7d413be85afb24e47c62ff2b/frontend-react/src/routes/Login/Login.jsx#L67-L70

**Review by H-isaac23 — CHANGES_REQUESTED** [review]
See comments

File: package-lock.json
Comment (JayAlvinCVallescas) [reply package-lock.json:1]: i delete ko yung package-lock.json?

Comment (H-isaac23) [reply package-lock.json:1]: Yes

**Review by H-isaac23 — APPROVED** [review]
lgtm

**Pull request merged**